docs/source/changelog.rst

```diff
@@ -30,6 +30,7 @@ Other Changes
 =============
 
 #. Removed temp partial movie file from SceneFileWriter (ffmpeg directly renders partial movie files instead of temp files)
+#. Improved handling of scenes without animations by detecting when no animations are present and outputting an image instead of a video 
```

Comment:
Thanks for adding your changes to the changelog.
But to avoid merge conflicts when continuously updating the cangelog.rst file, we decided to write most recent changes into the wiki: https://github.com/ManimCommunity/manim/wiki/Changelog-for-next-release. The cangelog.rst file will then be updated once for each release by the manim community devs.

Reply:
Just as @kolibril13 explained: please revert your changes to `changelog.rst`, it currently also produces a merge conflict.